rotkehlchen/chain/ethereum/modules/eth2/decoder.py

```diff
@@ -121,62 +127,113 @@ def _decode_eth2_deposit_event(self, context: DecoderContext) -> DecodingOutput:
         )
         return DEFAULT_DECODING_OUTPUT
 
-    def _decode_eth2_consolidation_request(self, context: DecoderContext) -> DecodingOutput:
-        """Decode a request to consolidate validators.
-        Handles two types of requests:
-        - Self consolidation - changes the withdrawal credentials to 0x02 (accumulating validator)
-          See https://epf.wiki/?ref=blog.obol.org#/wiki/pectra-faq?id=q-i-have-a-validator-with-0x01-credentials-how-do-i-move-to-0x02
-        - Normal consolidation - consolidates the source validator into the target validator
+    def _decode_eth2_request_with_fee(
+            self,
+            context: DecoderContext,
+            expected_data_length: int,
```

Comment:
perhaps make it a literal type
```suggestion
            expected_data_length: Literal[76, 116],
```